pyproject.toml

```diff
@@ -90,3 +248,6 @@ log_cli = true
 log_level = "DEBUG"
 log_cli_format = "%(asctime)s.%(msecs)03d %(levelname)5s --- [%(threadName)12s] %(name)-26s : %(message)s"
 log_cli_date_format = "%Y-%m-%dT%H:%M:%S"
+
+[tool.pip-tools]
+unsafe-package = ["localstack-core", "pip", "setuptools", "distribute"] # packages that should not be pinned
```

Comment:
A comment here would be great, since this looks a bit weird:
- Why is there pip, setuptools, distribute (because it's the default)
- Why is there localstack-core (because of the extra stacking)